### PR TITLE
C2LC-398: Fixed hover styles for share button in light theme.

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [C2LC-398](https://issues.fluidproject.org/browse/C2LC-398) for details.

### CODE REVIEW

File: src/Themes.scss [hunk 1]
```diff
@@ -296,8 +296,10 @@ body.light-theme {
     }
 
     button.btn.ShareButton {
-        background-color: #F1F2F4;
         color: black;
+        &:not(:hover) {
```
Comment (chosww) [inline src/Themes.scss:300]: Setting more specific rule for the base background-color seems not a good approach. We could have less specific rule for the ShareButton, so hover style can be applied from ShareButton component -- removing button.btn in front of the .ShareButton and setting background-color how it was before would fix this issue.

Comment (chosww) [reply src/Themes.scss:300]:
```
    .ShareButton {
        color: black;
        background-color: #F1F2F4;
    }
```

**Comment by chosww** [comment]
I also noticed that share button's hover colour seems not right in the grayscale theme. I assume that the correct colour should be the darker hover colour on other components, but would be if we could ask Sepideh to confirm that.

File: src/Themes.scss (hunk 1 above)
Comment (the-t-in-rtf) [reply src/Themes.scss:300]: I fixed this instance of it, but we still have lots of examples from the earlier theme work.  We can discuss a wider CSS cleanup in another ticket, or do it incrementally as we improve other elements.

**Comment by the-t-in-rtf** [comment]
I used the hover styles for the share button in the grayscale theme to match the theme selector, which looks reasonable.  Feel free to share with Sepi to confirm.

File: src/ShareButton.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 @import '_variables';
 
-button.btn.ShareButton {
```
Comment (chosww) [inline src/ShareButton.scss:3]: We still need button.btn part to apply the default interaction styles to the light theme. With this removed,the  light theme has no hover style.

Comment (the-t-in-rtf) [reply src/ShareButton.scss]: Good catch, and fixed.

**Comment by chosww** [comment]
Thanks @the-t-in-rtf, the changes look good.

**Pull request merged**